ddl/db_partition_test.go

```diff
@@ -160,14 +160,15 @@ func (s *testIntegrationSuite9) TestCreateTableWithPartition(c *C) {
 	);`)
 	c.Assert(ddl.ErrNotAllowedTypeInPartition.Equal(err), IsTrue)
 
-	sql9 := `create TABLE t9 (
-	col1 int
-	)
-	partition by range( case when col1 > 0 then 10 else 20 end ) (
-		partition p0 values less than (2),
-		partition p1 values less than (6)
-	);`
-	assertErrorCode(c, tk, sql9, tmysql.ErrPartitionFunctionIsNotAllowed)
+	// this sql is not allowed in mysql because of the result is const
+	//sql9 := `create TABLE t9 (
+	//col1 int
+	//)
+	//partition by range( case when col1 > 0 then 10 else 20 end ) (
+	//	partition p0 values less than (2),
+	//	partition p1 values less than (6)
+	//);`
+	//assertErrorCode(c, tk, sql9, tmysql.ErrPartitionFunctionIsNotAllowed)
```

Comment:
Why comment out this test? The error is changed?

Reply:
Updated, thanks